spec.html

```diff
@@ -43016,30 +42997,28 @@ <h1>Block-Level Function Declarations Web Legacy Compatibility Semantics</h1>
       <p>If an ECMAScript implementation has a mechanism for reporting diagnostic warning messages, a warning should be produced when code contains a |FunctionDeclaration| for which these compatibility semantics are applied and introduce observable differences from non-compatibility semantics. For example, if a var binding is not introduced because its introduction would create an early error, a warning message should not be produced.</p>
       <emu-annex id="sec-web-compat-functiondeclarationinstantiation">
         <h1>Changes to FunctionDeclarationInstantiation</h1>
-        <p>During FunctionDeclarationInstantiation the following steps are performed in place of step 29:</p>
+        <p>During FunctionDeclarationInstantiation the following steps are performed in place of step 28:</p>
```

Comment:
This appears to still be 29 in the rendering I'm looking at.

Reply:
Yup. (Was probably 28 at some point.) I'm fixing that, and also the internal step-references in EvalDeclarationInstantiation.